config/hub/samples/clustermanager_v1alpha1_multiclusterworkload.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: clustermanager.onprem.openshift.io/v1alpha1
+kind: MultiClusterWorkload
```

Comment:
This name was just an example we were using and is okay for now. We should consider coming up with better names at some point.

Reply:
Yes, I agree with it.